src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java

```diff
@@ -934,7 +934,11 @@ public boolean verifyVisibilityMessageIsDisplayed(int questionNumber) {
     public boolean verifyVisibilityOptionsIsDisplayed(int questionNumber) {
         return getVisibilityOptions(questionNumber).isDisplayed();
     }
-    
+
+    public WebElement getVisibilityOptionTableRow(int questionNumber, int optionRowNumber) {
+        return getVisibilityOptions(questionNumber).findElement(By.xpath("table/tbody/tr[" + optionRowNumber + "]"));
```

Comment:
This is fine, but if it's possible, in future, try to find a selector that doesn't depend on multiple nesting levels.